.swiftlint.yml

```diff
@@ -1,110 +1,106 @@
-# Compiled with swiftlint 0.13.0 rules
+included: # paths to include during linting.
+  - Source
```

Comment:
I actually think the `included` key should be removed entirely. If it's included then it must be changed by the project owner before running any swift lint commands. If the `included` key isn't in this file, then all paths (minus the paths defined in `excluded`) are marked for linting.

The difference here is really that this lint file doesn't work by default as-is unless you remove, or change the `included` key. This means the project owner needs to change it, AND Jenkins can't just grab this file for projects that are missing it (without some processing first).